generators/common/templates/README.md.ejs

```diff
@@ -298,7 +298,7 @@ If you need to re-run the Sonar phase, please be sure to specify at least the `i
 or
 <%_ } else if(buildTool === 'gradle') { _%>
 ```
-./gradlew -Pprod clean check sonarqube
+./gradlew -Pprod clean check jacocoTestReport sonarqube
```

Comment:
Will test it later on, but currently the binary jacoco data is generated always, so we could configure the build generate the xml report everytime, but not sure if thats a good idea.